### PR TITLE
Fix DropdownMenu out of range value

### PR DESCRIPTION
**Pull request by Jupi007**
Awaiting for fetching the values from Gsettings itself, here is a temporal patch for out of range value in `DropdownMenu`.

### CODE REVIEW

**Comment by Feichtmeier** [comment]
Nice thanks!

I see you also fixed a similar issue in the power page?

**Comment by Jupi007** [comment]
Yes, as all values are hardcoded, this problem can also happen on other dropdown if an out of range value is defined.

**Review by Feichtmeier — APPROVED** [review]

File: lib/view/pages/power/suspend_section.dart [hunk 1]
```diff
@@ -31,7 +31,9 @@ class _SuspendSectionState extends State<SuspendSection> {
         YaruRow(
           trailingWidget: const Text('Power Button Behavior'),
           actionWidget: DropdownButton<PowerButtonAction?>(
-            value: model.powerButtonAction,
+            value: PowerButtonAction.values.contains(model.powerButtonAction)
```
Comment (jpnurmi) [inline lib/view/pages/power/suspend_section.dart:34]: how about hiding this detail in the model so the view doesn't need to care?

Comment (Feichtmeier) [reply lib/view/pages/power/suspend_section.dart:34]: Paul just made a temp fix, a long term solution could be to not store the possible options in the model but get the list of options in a getter (I think - WDYT?)

Comment (Jupi007) [reply lib/view/pages/power/suspend_section.dart:34]: Even if it's a temp fix, I prefer to move this to the correct place :)
Thanks @jpnurmi

**Pull request merged**